### PR TITLE
Remove old fields from admin reports

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?233722#1195649

@czue

### CODE REVIEW

File: corehq/apps/hqadmin/reports.py [hunk 1]
```diff
@@ -750,22 +744,22 @@ def get_from_stat_facets(prop, what_to_get):
...
+            10: "cp_n_cases",
+            11: "cp_n_forms",
+            12: "cp_n_forms_30_d",
+            16: "cp_n_web_users",
+            28: "cp_n_out_sms",
+            29: "cp_n_in_sms",
+            30: "cp_n_sms_ever",
+            31: "cp_n_sms_in_30_d",
+            32: "cp_n_sms_out_30_d",
+            33: "cp_j2me_90_d_bool",
```
Comment (emord) [inline corehq/apps/hqadmin/reports.py]: This makes me sad every time

File: corehq/apps/domain/models.py [hunk 2]
```diff
@@ -146,7 +146,7 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
         choices=['', "plus", "community", "standard", "pro", "advanced", "enterprise"],
         default="community"
     )
-    services = StringProperty(choices=["", "basic", "plus", "full", "custom"], default="")
+    services = StringProperty(choices=["", "basic", "plus", "full", "custom"], default="")  # deprecated
```
Comment (czue) [inline corehq/apps/domain/models.py]: any reason not to just remove these?

File: corehq/apps/hqadmin/reports.py (hunk 1 above)
Comment (czue) [inline corehq/apps/hqadmin/reports.py]: wow this is amazing

File: corehq/apps/domain/models.py (hunk 2 above)
Comment (emord) [inline corehq/apps/domain/models.py]: I was a little scared that these might be used elsewhere. `date` and `services` are pretty hard to search for. I can remove them in a follow up and hope tests pass though

Comment (czue) [inline corehq/apps/domain/models.py]: :+1: that makes sense. no strong feelings

**Pull request merged**